src/main/java/seedu/duke/venue/VenueList.java

```diff
@@ -20,4 +20,12 @@ public static void printVenueInformation() throws EmptyListException {
             System.out.println(venueList.get(i));
         }
     }
+
+    public Venue getVenue(int venueNum) {
+        return venueList.get(venueNum);
+    }
+
+    public int getVenueListSize(){
```

Comment:
perhaps a space can be added between `)` and `{` for the codestyle to be consistent throughout